content/en/docs/contributing/localization.md

```diff
@@ -18,22 +18,44 @@ guidance offered in this section.
 
 ### Summary / TL;DR {#summary}
```

Comment:
Noticed this and I'm not that sure that folks will get what `TL;DR` is. I'd remove it, honestly.